utils.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+function convertUrlToWebSocket (urlString) {
+  return urlString.replace(/(http)(s)?:\/\//, 'ws$2://')
```

Comment:
what about a more robust one which tests only the beginning of the string?

Reply:
👍